okrs/2019-q1.md

```diff
@@ -6,29 +6,29 @@ Background:
 
 | Key Result | Assignee | Priority | Mid-Q Score | Projected Score | Final Score |
 | ---------- | -------- | -------- | ----------- | --------------- | ----------- |
-| IPLD Selector grammar can be parsed into an AST w/ support for basic grammar in Go. | @warpfork | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
-| IPLD Selector grammar can be parsed into an AST w/ support for basic grammar in JS. | @vmx | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
-| A DAG can be traversed to produce blocks matching a specified Selector AST in Go. | @warpfork | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
-| A DAG can be traversed to produce blocks matching a specified Selector AST in JS. | @vmx | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | | | |
+| IPLD Selector grammar can be parsed into an AST w/ support for basic grammar in Go. | @warpfork | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | 0.5 | 1.0 | |
+| IPLD Selector grammar can be parsed into an AST w/ support for basic grammar in JS. | @vmx | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | 0.5 | 1.0 | |
+| A DAG can be traversed to produce blocks matching a specified Selector AST in Go. | @warpfork | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | 0.5 | 1.0 | |
+| A DAG can be traversed to produce blocks matching a specified Selector AST in JS. | @vmx | ![p0](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p0.svg) | 0.5 | 1.0 | |
 
 ## Objective: Interfaces are in-line with current IPLD specifications and vision.
 
 | Key Result | Assignee | Priority | Mid-Q Score | Projected Score | Final Score |
 | ---------- | -------- | -------- | ----------- | --------------- | ----------- |
-| `go-ipld-prime` is a usable replacement for `go-ipld-cbor`. | @warpfork | ![p1](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p1.svg) | | | |
-| `js-ipld-interace` refactoring is complete. | @vmx | ![p1](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p1.svg) | | | |
-| Multiformat API's are easier to use with IPLD. | @vmx | ![p2](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p2.svg) | | | |
-| `ipfs-block` API is moved to IPLD project.| @mikeal | ![p2](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p2.svg) | | | |
+| `go-ipld-prime` is a usable replacement for `go-ipld-cbor`. | @warpfork | ![p1](https://ipfs.io/ipfs/QmV88khHDJEXi7wo6o972MZWY661R9PhrZW6dvpFP6jnMn/p1.svg) | 0.6 | 0.5 | |
```

Comment:
hrm... are you planning to delete some of the work you completed @warpfork? :)

Reply:
Hopefully not!  I was ... reading that as projections I would've made... earlier.  Are they supposed to be midq projections for eoq?  xD

Reply:
Yup :) It’s a way to surface early if we don’t think something will take longer than we originally anticipated.